docs/books/admin_guide/09-backups.md

```diff
@@ -125,13 +125,13 @@ The commands we will use here are `tar` and `cpio`. If you want to learn about t
 
 !!! Note
 
-    These commands save in a proprietary and standardized format.
+    These commands are saved in a proprietary and standardized format.
```

Comment:
Welcome back @gannazhyrnova! I think this change introduces passive voice. The original language is active voice, is it OK to save that in this case?